Fix git rpr

emilevauge opened the pull request:
This PR adds an argument to manage upstream correctly: `git rpr pr remote/branch`
It also adds `git rmpr` & `git cpr` commands.

/cc @vdemeester 

Signed-off-by: Emile Vauge <emile@vauge.com>

vdemeester commented on .github/rpr.sh:
```diff
@@ -4,9 +4,9 @@
+usage="$(basename "$0") pr remote/branch -- rebase a Pull Request against a remote branch"
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -ne 2 ]; then
```
Should we have a default value for the most common case ? (if `$2` is not present, use `upstream/master` for ex 👼 )

emilevauge commented:
@vdemeester I added 1/. On 2/, is it really needed? The script will fail if the specified remote doesn't exist, right ?

vdemeester commented on .github/cpr.sh:
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# git config --global alias.rpr '!sh .github/cpr.sh'
```
This alias is not right 👼

vdemeester commented on .github/rmpr.sh:
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# git config --global alias.rpr '!sh .github/rmpr.sh'
```
This alias is not right 👼 👼

emilevauge commented:
@vdemeester 
>It will, but it leaves you in a weird state (not on the same branch when you ran the script initially).
Added a TRAP calling clean.

>Also, shouldn't rpr use cpr and rmpr ?
Indeed, fixed.
